tasks/skylab/SequenceDataWithMoleculeTagMetrics.wdl

```diff
@@ -2,22 +2,22 @@ version 1.0
 
 task CalculateGeneMetrics {
   input {
-    File bam_input
+    File tsv_input
 
     # runtime values
-    String docker = "quay.io/humancellatlas/secondary-analysis-sctools:v0.3.11"
+    String docker = "quay.io/humancellatlas/secondary-analysis-python3-scientific:sctools-optimized"
```

Comment:
Will be updated to a released sctools docker before merge, right?

Reply:
updated